src/components/Home/Aditya/Week4/pages/Week3/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const statusChoices = ['Busy', 'Active', 'Do not disturb', 'Idle'];
```

Comment:
you can use the array which keshav had explicitly made to reduce the code redundancy instead of making new array file